.github/SECURITY.md

```diff
@@ -2,30 +2,69 @@
 
 ## Supported Versions
 
-We support these versions of AzerothCore:
+We support the following versions of dependencies.
+
+:white_check_mark: = supported
+:red_circle: = NOT supported
```

Comment:
We dont need to tell them what we don't support, anything that is not supported is just automatically not supported

Reply:
@Kitzunu this is to be more explicit about things we know for sure that won't work at all (or are strongly discouraged to use). There are some versions that we don't check in our CI but they still probably work, we need to make a dinstinction

Reply:
Well as for DB Versions at least, we don't need to specify what we do not support because for example running the server without MySQL 5.7 or above is impossible

Reply:
we do as we explicitly want to tell the user that a lower version should NOT be used, to distinguish cases were we just can't guarantee it works but there is nothing wrong about trying it, e.g. ubuntu 20.10